nss-day-cohort-45/tabloid-cli-nights-court
# Koh delete blog

## Changes
1. Updated the Delete method to Update the isDeleted column rather than remove the blog from the database
2. Updated GetAll and Get methods in BlogRepository to show blogs where isDeleted = 0

## To Test
1. Fetch --all and open branch (kohDeleteBlog)
2. Add a new blog and then add a post associated with the new blog.
3. Remove the blog  you just added (option 5)
4. List all the blogs to ensure it's not visible. Also check Blog Details, Add Blog, Edit Blog, and Remove Blog to be sure it's not on those lists. 
5. Check the post you just created. Be sure the post is still visible.
6. Run a query - SELECT * FROM Blog; (You should also see the updated isDeleted column for the blog you removed.)
